plugins/forc-index/src/commands/build.rs

```diff
@@ -42,10 +42,10 @@ pub struct Command {
     #[clap(long, help = "Building for native execution.")]
     pub native: bool,
 
-    /// Path with which to prefix asset filepaths in the index manifest.
+    /// Path with which to prefix asset filepaths in the indexer manifest.
     #[clap(
```

Comment:
this comment / help has always confused me. Suggested change (or something similar) 
`/// output path of indexer assets in the manifest`

Reply:
@rvmelkonian I agree this is confusing. It's actually cleaned up in https://github.com/FuelLabs/fuel-indexer/pull/676 (so don't mind it here)